packages/tools/package/src/models/package/package.ts

```diff
@@ -152,12 +160,21 @@ class Package {
    * @returns - Promise resolving with this Package instance.
    */
   public inspect(): Promise<this> {
-    return Yarn.view({ package: this.data.name, distTags: true })
-      .then((tags) => {
-        const version = tags[this.data.version.tag]
-          || `${tags[CONSTANTS.STABLE_TAG] || '0.0.0'}-${this.data.version.tag}.0`;
+    const { tag } = this.data.version;
+
+    return Package.inspect({ package: this.data.name })
```

Comment:
Just curious to understand this. Why are we using the static Package for these calls?